Quick_Deploy/vLLM/model_repository/vllm/1/model.py

```diff
@@ -166,16 +166,21 @@ async def generate(self, request):
         self.ongoing_request_count += 1
         try:
             request_id = random_uuid()
-            prompt = pb_utils.get_input_tensor_by_name(request, "PROMPT").as_numpy()[0]
+
+            prompt = pb_utils.get_input_tensor_by_name(request, "text_input").as_numpy()[0]
             if isinstance(prompt, bytes):
                 prompt = prompt.decode("utf-8")
-            stream = pb_utils.get_input_tensor_by_name(request, "STREAM").as_numpy()[0]
+
+            # stream is an optional input
+            stream = False
```

Comment:
nit: Would it be better for this to be defined as a function parameter?
```python
async generate(self, request, stream=False):
    ...
```

Reply:
might be confusing as its not really an input  / would get override by the request.

@rmccorm4 do you have a preference?

Reply:
I think it's more intuitive to keep it close to the input tensor parsing where it is now, it would be unintuitive to overwrite the arg later.